src/harness/harness.c

```diff
@@ -264,13 +268,34 @@ void Harness_Hook_BrBufferUpdate(br_pixelmap* pm, br_token use, br_uint_16 flags
     }
 }
 
+// Input hooks
+void Harness_Hook_GetMousePosition(int* pX, int* pY) {
```

Comment:
What if we made this `Harness_Hook_GetRelativeMousePosition`, and has this function to the translation to window co-ordinates? Then we could avoid exposing also `Harness_Hook_GetWindowSize`?